docs/tasks/administer-cluster/kubeadm-upgrade-1-9.md

```diff
@@ -0,0 +1,275 @@
+---
+approvers:
+- pipejakob
+- luxas
+- roberthbailey
+- jbeda
+title: Upgrading kubeadm clusters from 1.8 to 1.9
```

Comment:
We now support downgrades as well; can we rephrase this to be sth like `Upgrading/downgrading kubeadm clusters between v1.8 and v1.9`?